MODULE.bazel

```diff
@@ -58,7 +58,7 @@ bazel_dep(name = "bazel_skylib_gazelle_plugin", version = "1.4.1", dev_dependenc
 bazel_dep(name = "stardoc", version = "0.6.2", dev_dependency = True, repo_name = "io_bazel_stardoc")
 
 go_sdk_dev = use_extension("@io_bazel_rules_go//go:extensions.bzl", "go_sdk", dev_dependency = True)
-go_sdk_dev.download(version = "1.22.4")
+go_sdk_dev.download(version = "1.22.5")
```

Comment:
@fmeum This one is fine though right? because we want to verify as large a _range_ of versions as possible with the dev dependency.

Reply:
Yes, that's a good idea.